### ISSUE TITLE
Goals of the website

### ISSUE THREAD

**Issue by wtimme** [bryceco/gomaposm.com#1]
Let's discuss the goals of the website.

1. Serve the `.well-known/apple-app-site-association` file
2. Allow for users to download _Go Map!!_ from the App Store
3. Allow for users to edit the map in iD

The last point important for desktop users who get a gomaposm.com link and want to make use of it.

What's your point of view here, @bryceco? @dieterdreist, do you have any input/expectations?

**Comment by dieterdreist** [comment]
as a very low hanging fruit, I would add 
`<meta name="apple-itunes-app" content="app-id=592990211, affiliate-data=myAffiliateData, app-argument=">
`
to the page header. "myAffiliateData" should be filled with actual data or removed (is optional).
Docs here https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html

This will create an install/open banner on iOS devices on the pages where it is added (can be dismissed "permanently" by the user until they delete their browsing history).

**Comment by wtimme** [comment]
Good idea! I've added the banner to the page. This is what it looks like now:

![smart-app-banner](https://user-images.githubusercontent.com/1681085/92588301-15548480-f299-11ea-83ff-e9cd1689031a.jpeg)

**Comment by dieterdreist** [comment]
regarding iD editing, this is blocked (not shown) on the osm website for phone-size devices (not sure if it works on tablets)

**Comment by dieterdreist** [comment]
we should change the logo, both on the website and in the app.

**Comment by dieterdreist** [comment]
there should also be a short text, maybe the one from the app description, in order to make the page findable by search engines. Keywords are openstreetmap, osm, editing, editor, iOS, iPhone, iPad, map, mobile editor, etc. 
Each of these should appear in the intro text, possibly more than once.

**Comment by dieterdreist** [comment]
I also got a warning about "insecure" elements, looking at the page source, the only http:// links I found are in the header doctype file:
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

```

try to change these 2 to http**s**://

**Comment by tordans** [comment]
I suggest using github sites for hosting the content, that would make the server and deployment part very easy.

**Comment by tordans** [comment]
We need an imprint on the site and other legally formal documents, like a privacy doc.

**Comment by dieterdreist** [comment]
Am Mi., 9. Sept. 2020 um 13:05 Uhr schrieb Tobias <notifications@github.com
>:

> We need an imprint on the site and other legally formal documents, like a
> privacy doc.
>


I am not sure, but I would not think so, this is a US hosted site, isn't it?

**Comment by bryceco** [comment]
For me the site exists only to host the universal link. I didn’t intend for the site to be visited in a browser necessarily, since the site containing the referring link can detect whether the app is installed by checking if gomaposm:// is a valid URI (as I understand things). 

If people want to make something more of the page that’s fine. But @wtimme’s #1 and #2 are the only items I’m concerned with.

**Comment by tordans** [comment]
The thing is, from a legal point of view it does not matter if the page only opens and immediately redirects. It's still a Datenverarbeitung / data processing. I don't thinks it a big deal and it's great to have the feature, but we should address it at some point.

**Comment by dieterdreist** [comment]
sent from a phone

> On 9. Sep 2020, at 19:34, Tobias <notifications@github.com> wrote:
> 
> It's still a Datenverarbeitung / data processing. I don't thinks it a big deal and it's great to have the feature, but we should address it at some point.


it’s a German (imprint) and EU (privacy) thing, it is not necessary for an US site (maybe something has to be added according to federal or state law, no idea).

**Comment by tordans** [comment]
> … and EU (privacy) thing, it is not necessary for an US site …

As I said, that't not my understanding and I read (eg.) https://gdpr.eu/compliance-checklist-us-companies/ the same way and https://wiki.openstreetmap.org/w/images/8/88/GDPR_Position_Paper.pdf as well.

**Comment by dieterdreist** [comment]
sent from a phone

> On 9. Sep 2020, at 21:25, Tobias <notifications@github.com> wrote:
> 
> As I said, that't not my understanding and I read (eg.) https://gdpr.eu/compliance-checklist-us-companies/ the same way and https://wiki.openstreetmap.org/w/images/8/88/GDPR_Position_Paper.pdf as well.


which personal data do you refer to in the context of gomap? 

Nobody has ever been processed outside of the EU, because although they claim that the scope goes beyond the jurisdiction, reality is that the effect remains within in EU.

**Comment by wtimme** [comment]
Ideally, we would not use an app-dedicated domain, but rather openstreetmap.org, as nobody except _Go Map!!_ users can use our links proper. (The gomaposm.com domain defeats the idea of the Universal Link, as far as I understand them. I would not send my Android buddy such a link, because they cannot download _Go Map!!_. Instead, I would probably share a openstreetmap.org link, since it is truely universal.)

I suggest we ask the openstreetmap.org maintainers to include the ‘.well-known‘ file so that users are able to visit openstreetmap.org links, which would then open _Go Map!!_. If our app was supporting all of the links (editing nodes/ways/relations, viewing objects/areas), we would be a true iOS native companion app.

The last time I asked about this, I got sceptic feedback ("Wait, we are supposed to include a file in order to support iOS?"), but I think it's worth it to ask again.

Users would be able to seamlessly open openstreetmap.org link on their iOS device, right in _Go Map!!_. I don't know about your use cases, but I would welcome such an integration.

**Comment by bryceco** [comment]
I agree that having the file on openstreetmap.org is the ideal case. I expect that a pull request that enabled "Edit on iOS/Android" and included the app association file would be accepted.
